db/migrations/20200108074134_create_vat_rely.sql

```diff
@@ -5,17 +5,14 @@ CREATE TABLE maker.vat_rely
     id         SERIAL PRIMARY KEY,
     header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
     log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
-    address_id INTEGER NOT NULL REFERENCES addresses (id) ON DELETE CASCADE,
```

Comment:
Are we removing the address_id from here (and from vat_deny) because there's only going to be one vat contract per release?

Reply:
Yeah exactly